lms/djangoapps/teams/static/teams/templates/edit-team.underscore

```diff
@@ -48,7 +75,7 @@
     <button class="action action-cancel">
         <%=
             interpolate_text(
-                _.escape(gettext("Cancel {span_start}a new team{span_end}")),
+                _.escape(gettext("Cancel {span_start} team {span_end}")),
```

Comment:
Ah ok, yeah the screenreader text suggested makes sense to me. (cc'ed @catong )

(Also, we can revisit our use of this semi-modal pattern moving forward, as it will increasingly impact more pages and part of the experience in future quarters. (ex. Create New Discussion Topic / Post, etc.)